dev-libs/libstrophe/libstrophe-0.12.0.ebuild

```diff
@@ -6,7 +6,7 @@ DESCRIPTION="A simple, lightweight C library for writing XMPP clients"
 HOMEPAGE="http://strophe.im/libstrophe/"
 SRC_URI="https://github.com/strophe/${PN}/releases/download/${PV}/${P}.tar.xz"
 LICENSE="|| ( MIT GPL-3 )"
-SLOT="0"
+SLOT="0/0.1"
```

Comment:
I'd add a comment explaining the value and that on next change it can be made something sensible like 1 or whatever the new SONAME is

with that, lgtm

Reply:
done